master/internal/command/tensorboard_manager.go

```diff
@@ -400,3 +405,15 @@ func getEnvVars(m map[string]string) []string {
 
 	return envVars
 }
+
+func refineArgs(s []string) {
+	trimmed := ""
+	for x := range s {
+		trimmed = strings.TrimLeft(s[x], "-")
+		if trimmed == "h" {
```

Comment:
question: should we be just dropping `-h` (it is to bring up the help menu right?)

Reply:
yes, `-h` and `--help` bring up the help menu. Should we drop it or just let it happen? My thought for arguments was just to let them happen.

Reply:
I would probably just drop it, running it with `-h` would just lead to the container hanging and timing out.